packages/xo-server-usage-report/src/index.js

```diff
@@ -759,6 +759,12 @@ class UsageReportPlugin {
   }
 
   async _sendReport(storeData) {
+    if (this._xo.sendEmail === undefined) {
+      throw new Error(
+        'The plugin usage-report requires the plugin transport-email to be enabled'
+      )
```

Comment:
Should it also unload the plugin?

Reply:
I don't think so. The user will see the error and enable the plugin transport-email then re-send the report

Reply:
Ok but what if it's an automatic usage report being sent? We probably don't want to retry next time and trigger more errors.